pkgs/by-name/ty/typstyle/package.nix

```diff
@@ -29,22 +24,6 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
-  nativeBuildInputs = [
```

Comment:
Why is this no longer required?

Reply:
OK, let's wait for the CI and see if it builds without all of this on Darwin.

Reply:
It does, I have run `nixpkgs-review` for all platforms.

Reply:
Oh. Great ! All good then.